app/src/main/res/values/styles_fonts.xml

```diff
@@ -107,6 +107,10 @@
         <item name="android:textSize">14sp</item>
     </style>
 
+    <style name="WalletTheme.TextAppearance.Medium12" parent="WalletTheme.TextAppearance.Medium">
```

Comment:
are we sure that we have a new style only for this new items? maybe it's a mistake?